benchmarks/torchbench_model.py

```diff
@@ -264,7 +264,9 @@ def set_up(self):
     benchmark = self.load_benchmark()
 
     self.module, self.example_inputs = benchmark.get_module()
-
+    if isinstance(self.example_inputs,
```

Comment:
Leave a comment here, mentioning the pytorch/benchmark PR. It would be good to know in the future.